source/Octopus.Client/Model/ProjectResource.cs

```diff
@@ -86,21 +86,6 @@ public ProjectResource(string id, string name, string slug) : this()
         [Writeable]
         public List<string> IncludedLibraryVariableSetIds { get; set; }
 
```

Comment:
I know we can't _really_ remove these, but a girl can dream, can't she?
What's the right approach here for maintaining backwards compat - obsolete tags?

Reply:
Yes, `Obsolete` and try and guide the consumer to the new way of doing things. I.e. Something like:

```
[Obsolete("Use `DeploymentSettingsResource` instead on the `deploymentsettings` API.")]
```